### ISSUE TITLE
performance of caption stage2

### ISSUE THREAD

**Issue by Tangshengku** [OFA-Sys/OFA#267]
Hi, thank you for your fantastic work!

I have some issues related to the performance change of the optimization in stage2 training of image caption.

In my stage2 training(some modification in stage1, but no modification during stage2), I do evaluation after every 2000 steps. There are 4721 steps in an epoch. My evaluation scores are 1.47, 1.435, 1.448. It seems that the best score is around the beginning of the training. Will this happen to you during your training? Or I should do some modification to reduce the performance drop?

Hope to have your reply!
Best regrads,
Shengkun Tang

**Comment by logicwong** [comment]
@Tangshengku CIDEr optimization is very sensitive to hyperparameters, you can try to reduce the learning rate and increase the batch size.

**Comment by Tangshengku** [comment]
> @Tangshengku CIDEr optimization is very sensitive to hyperparameters, you can try to reduce the learning rate and increase the batch size.

Thank you for your fast reply. I will try to reduce the learning rate and increase the batch size to show the performance. I will report the results in this page. 
By the way, with limited GPU  memory, increasing update_freq is the same as increasing the batch size for performance, right?

Shengkun Tang

**Comment by logicwong** [comment]
> increasing

Yes, increasing update_freq is the same as increasing the batch size. One more suggestion, using `--freeze-resnet` can improve the training stability.

**Comment by Tangshengku** [comment]
> @Tangshengku Yes, increasing update_freq is the same as increasing the batch size. One more suggestion, using `--freeze-resnet` can improve the training stability.

Thank you for your suggestion~ I will report the results here later if it is OK and close the issues.

Shengkun Tang
